designs/node-ownership.md

```diff
@@ -0,0 +1,224 @@
+# Karpenter Node Ownership
+
+*NOTE: Provisioning is used throughout this doc to reference Karpenter scheduling and spinning up a node. Note that this can apply to both the pending pod scheduling loop and also the consolidation scheduling loop. Essentially, any scheduling loop where Karpenter launches pods.*
```

Comment:
🤷 
```suggestion
Provisioning is used throughout this doc to reference Karpenter scheduling and spinning up a node. This can apply to both the pending pod scheduling loop and also the consolidation scheduling loop. Essentially, any scheduling loop where Karpenter launches pods.
```